src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -493,6 +487,20 @@ components:
         - steps
         - prisonId
 
+    CreateGoalsRequest:
+      title: CreateGoalsRequest
+      type: object
+      description: A request to create one or more Goals that a Prisoner aims to complete as part of their Action Plan. The goal's status will be set to 'ACTIVE'.
```

Comment:
Minor (and pedantic I know, sorry!):

Perhaps change "The goal's status will be..." (which implies singular) to "The status of each goal will be...".

Reply:
ok 👍

Reply:
Done